tool/tsh/common/tsh.go

```diff
@@ -801,6 +811,14 @@ func Run(ctx context.Context, args []string, opts ...CliOption) error {
 	ssh.Flag("log-dir", "Directory to log separated command output, when executing on multiple nodes. If set, output from each node will also be labeled in the terminal.").StringVar(&cf.SSHLogDir)
 	ssh.Flag("no-resume", "Disable SSH connection resumption").Envar(noResumeEnvVar).BoolVar(&cf.DisableSSHResumption)
 	ssh.Flag("relogin", "Permit performing an authentication attempt on a failed command").Default("true").BoolVar(&cf.Relogin)
+	// The following flags are OpenSSH compatibility flags. They are used for
+	// users that alias "ssh" to "tsh ssh." The following OpenSSH flags are
+	// implemented. From "man 1 ssh":
+	//
+	// * "-V Display the version number and exit."
+	// * "-T Disable pseudo-terminal allocation."
+	ssh.Flag(uuid.New().String(), "").Short('T').Hidden().BoolVar(&cf.NonInteractive)
```

Comment:
Tiny nit, but on second thought, can we just hardcode a UUID (or some other long and unlikely-to-ever-be-used flag name) here?

Generating a new UUID is cheap, but it's not free, and this code runs during initialization of tsh (for every single invocation).

Reply:
Hrm, interesting question. I used `hyperfine` to see if there is a difference.

With `uuid.New().String()`:

```
$ hyperfine -w 10 -m 100 "./tsh.sh ssh -V"
Benchmark 1: ./tsh.sh ssh -V
  Time (mean ± σ):      60.7 ms ±   3.3 ms    [User: 41.6 ms, System: 17.6 ms]
  Range (min … max):    50.8 ms …  71.9 ms    100 runs
```

With a hardcoded flag name:

```
$ hyperfine -w 10 -m 100 "./tsh.sh ssh -V"
Benchmark 1: ./tsh.sh ssh -V
  Time (mean ± σ):      60.9 ms ±   3.0 ms    [User: 41.6 ms, System: 17.2 ms]
  Range (min … max):    56.3 ms …  74.0 ms    100 runs
```

I can't explain why it's faster using `uuid.New().String()` other than it isn't, we're just seeing noise from something else running on the system.

That being said, if you guys feel strongly about this, I can change it to a hardcoded string.